Tests/configure_and_test_integration_instances.py

```diff
@@ -45,6 +45,7 @@
 NO_PROXY = ','.join([
     'oproxy.demisto.ninja',
     'oproxy-dev.demisto.ninja',
+    '*.sharepoint.com'
```

Comment:
What is this? Why it is needed?

Reply:
We are experiencing SSL connection errors in the playbook, this is the fix suggested by @esharf